config/300-revision.yaml

```diff
@@ -38,6 +38,9 @@ spec:
   - name: Service Name
     type: string
     JSONPath: .status.serviceName
+  - name: Age
```

Comment:
This is more like `birthdate`, than age.

Reply:
I believe kubectl renders all timestamps as age and I hate it.

Reply:
Yeah - I was being consistent with K8s